.github/workflows/stg_web_svc_merge.yml

```diff
@@ -10,17 +10,15 @@ env:
   NODE_VERSION: "20"
 
 jobs:
-
   web_cd:
     name: Deploy web service
     runs-on: ubuntu-latest
 
     env:
-      ENV_NAME              : stg
```

Comment:
I don't see any relevant changes to the issue; it's only formatting. Can you revert these changes?

Reply:
yes

Reply:
As long as I hit cmd+s in VSCode, it's reformatted automatically. I'm not sure how to resolve it.